tests/operator_test.go

```diff
@@ -1284,6 +1306,8 @@ spec:
 			curVersion := originalKv.Status.ObservedKubeVirtVersion
 			curRegistry := originalKv.Status.ObservedKubeVirtRegistry
 
+			curOperatorManifestPath := filepath.Join(flags.ManifestsDir, "release/kubevirt-operator.yaml")
```

Comment:
This `release/*.yaml` addition here makes it a little bit hard to point to the operator yaml. Maybe directly make a commandline flag which points to the operator yaml.

Reply:
Changed in 9d3dd75.